api/tests/test_api.py

```diff
@@ -234,3 +235,53 @@ async def test_push_points():
             assert response.status_code == 200
             assert mp_unpackb(response._content) == "data sent"
         del ps
+
+
+class TrialA(BaseModel):
+    integers: list[int]
```

Comment:
Capitalise list type as `List`

Reply:
New to [Python 3.9](https://docs.python.org/3.11/whatsnew/3.9.html#type-hinting-generics-in-standard-collections) is [PEP 585](https://peps.python.org/pep-0585) that allows type hints in generics with standard collections.